docs/libs/gelly_guide.md

```diff
@@ -734,6 +737,24 @@ public static final class Apply {
 
 {% endhighlight %}
 
+The following example illustrates the usage of the edge direction option. Vertices update their values to contain a list of all their in-neighbors.
```

Comment:
What do you mean by "contain a list of all their in-neighbors"? Were you planning to add an example where vertices assign their neighborhoods as their values?